presto-verifier/src/main/java/com/facebook/presto/verifier/Validator.java

```diff
@@ -253,7 +316,7 @@ public QueryResult getTestResult()
         return testResult;
     }
 
-    private QueryResult executeQuery(String url, String username, String password, Query query, Duration timeout, Map<String, String> sessionProperties)
+    private QueryResult executeQuery(String url, String username, String password, Query query, String sql, Duration timeout, Map<String, String> sessionProperties)
```

Comment:
I would have both the old method `private QueryResult executeQuery(String url, String username, String password, Query query, Duration timeout, Map<String, String> sessionProperties)` and this one. Then move all the logic to run the pre and post queries into that old method signature, and have it use this one.